roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/kargs.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Personalize template
+  ansible.builtin.template:
+    src: 99-kargs-worker.yml.j2
+    dest: "{{ __sap_hypervisor_node_preconfigure_register_tmpdir.path }}/99-kargs-{{ __sap_hypervisor_node_preconfigure_register_worker_name }}.yml.j2"
+    mode: "0644"
+
+- name: Enable hugepages
+  kubernetes.core.k8s:
+    state: present
+    src: "{{ __sap_hypervisor_node_preconfigure_register_tmpdir.path }}/99-kargs-{{ __sap_hypervisor_node_preconfigure_register_worker_name }}.yml.j2"
```

Comment:
Use template directly here, why is a separate call to the `template` module necessary?

Reply:
Because rendering of the template not happening otherwise :-S